README.md

```diff
@@ -8,7 +8,7 @@ DF-structures is the core information on which
 For information on how to understand this format,
 see [./SYNTAX.rst](./SYNTAX.rst).  For how to update
 for a new version of DF, see
-[./how-to-update.rst](./how-to-update.rst)
+[new-release-processing.rst](https://github.com/DFHack/df_misc/blob/master/doc/new-release-processing.rst)
```

Comment:
Which docs build? CI broke briefly because there was a reference in dfhack to the removed file, but that was fixed as soon as both PRs were in. 

Or do you mean the PDF build?